lib/ae_mdw/auction_bids.ex

```diff
@@ -95,6 +109,37 @@ defmodule AeMdw.AuctionBids do
   end
 
   defp render(
+         state,
+         Model.auction_bid(
+           index: plain_name,
+           block_index_txi_idx: {block_index, _txi_idx},
+           expire_height: expire_height,
+           bids: [last_bid | _rest_bids]
+         ),
+         last_gen,
+         last_micro_time,
+         _opts
+       ) do
+    last_bid = Txs.fetch!(state, bi_txi_idx_txi(last_bid))
+    name_ttl = Names.expire_after(expire_height)
+
+    %{
+      name: plain_name,
+      activation_time: DbUtil.block_index_to_time(state, block_index),
+      auction_end: expire_height,
+      approximate_expire_time:
+        DbUtil.height_to_time(state, expire_height, last_gen, last_micro_time),
+      last_bid: put_in(last_bid, ["tx", "ttl"], name_ttl)
```

Comment:
let's remove the `tx_index` from all responses?

Reply:
Good call, done